pages/toDOH.jsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from "react";
+import Header from "../components/Header";
+import ToDOHModal from "../components/Modal";
+import { IconButton } from "@chakra-ui/button";
+import { Flex } from "@chakra-ui/layout";
+import { SunIcon, PhoneIcon } from "@chakra-ui/icons";
+
+const toDOHs = [
+  {
+    id: 1,
+    name: "Male das Pony",
+    inside: true,
+    description1:
+      "Hol dir ein Blatt Papier und einen Stift und male ein Pony, das eine Möhre Kaut und einen Hut auf hat",
+    description2: "Setze deinen Handy-Timer auf 15 Minuten",
+  },
+  {
+    id: 2,
+    name: "Putze dein Küchenfenster",
+    inside: true,
```

Comment:
i'd suggest using `isInside` to make clear that `inside` is in fact a boolean value